### DISCUSSION TITLE
Keep latest updates after mirror

### DISCUSSION THREAD

**Discussion by ouikhuan** [community/community#22921] [New to GitHub]
<p>Hi all,</p>
<p>I’m trying to migrate one of my repo to github and following the instructions below:</p>
<p><a href="https://help.github.com/articles/importing-a-git-repository-using-the-command-line/" rel="nofollow noopener">https://help.github.com/articles/importing-a-git-repository-using-the-command-line/</a></p>
<p>And everything works great! However…after I did the push mirror and remove the local repo like step 4, there’s some updates from my old repo (original repo), is there a way to sync it over to the new repo on github again?</p>
<p>Thanks!</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/wecanwang">@wecanwang</a>,</p>
<p>You can either <a href="https://help.github.com/articles/about-github-mirrors/" rel="nofollow noopener">set up a post-receive hook on the remote repository that you duplicated to push new changes</a> or <a href="https://help.github.com/articles/syncing-a-fork/" rel="nofollow noopener">update your downstream repository manually</a> to get things in sync.</p>
<p>Thanks!</p>

**Comment by ouikhuan** [comment]
<p>Thanks for answering the question <a class="mention" href="/that-pat">@that-pat</a>. </p>
<p>I’ll try to setup a upstream to see if I can update it manually!</p>
